api/services/ocean/aws/paths/controllerHeartbeat.yaml

```diff
@@ -8,7 +8,14 @@ get:
   - "Ocean AWS"
   parameters:
   - $ref: "../../../../commons/parameters/accountId.yaml"
-  - $ref: "../../commons/parameters/oceanId.yaml"
+  - in: "path"
```

Comment:
@vlad666spot why changing?

Reply:
I experience issue with path param Ocean ID and made it inline - but ref is applicable so I change it back. Fixed